docs/en/testing/supplemental.md

```diff
@@ -100,23 +100,23 @@ Consider the following widget which displays an additonal `h2` when the `LG` bre
 
 > src/Breakpoint.tsx
 
-```
+```tsx
 import { tsx, create } from '@dojo/framework/core/vdom';
 import breakpoint from '@dojo/framework/core/middleware/breakpoint';
 
 const factory = create({ breakpoint });
 
 export default factory(function Breakpoint({ middleware: { breakpoint } }) {
-  const bp = breakpoint.get('root');
-  const isLarge = bp && bp.breakpoint === 'LG';
-
-  return (
-    <div key="root">
-      <h1>Header</h1>
-      {isLarge && <h2>Subtitle</h2>}
-      <div>Longer description</div>
-    </div>
-  );
+	const bp = breakpoint.get('root');
```

Comment:
how come this was reformatted?

Reply:
@agubler This was the result of running prettier against the markdown files. I believe that example was using spaces instead of tabs and Prettier corrected it.

Reply:
Oh weird running prettier against master doesn't do anything for me.